src/components/visual-editor/index.jsx

```diff
@@ -89,7 +91,11 @@ function VisualEditor( { hideTitle } ) {
 		// context.
 		commonStyles,
 		// Add sensible default styles if theme styles are not present.
-		hasThemeStyles ? '' : defaultThemeStyles
+		hasThemeStyles ? '' : defaultThemeStyles,
+		componentStyles,
+		blockEditorContentStyles,
+		blocksStyles,
+		blocksEditorStyles
```

Comment:
Relocated from `utils/editor-styles` to scope their loading to the block canvas `iframe`, per the [Gutenberg Framework guides](https://wordpress.org/gutenberg-framework/docs/intro#bootstrap-your-block-editor).